File: AarPluginForUnity/ActivityCheckPlugin/src/main/java/com/showcase/activitycheckplugin/StartActivityHelper.java
package com.showcase.activitycheckplugin;

import android.content.Context;
import android.content.Intent;

public class StartActivityHelper {

    public static void openActivity(Context context) {
        Class needActivity = ActivityStringResources.class;
        Intent intent = new Intent(context, needActivity);
        context.startActivity(intent);
    }
}

File: AarPluginForUnity/ActivityCheckPlugin/src/main/java/com/showcase/activitycheckplugin/StartActivityHelper.kt
package com.showcase.activitycheckplugin

import android.content.Context
import android.content.Intent

object StartActivityHelper {                                                // синглтон
    fun openActivity(context: Context) {

        var needActivity = ActivityStringResources::class.java;             // берем ссылку на активити которую хотим запустить
        val intent = Intent(context, needActivity)                          // обертка для открытия активити
        context.startActivity(intent)                                       // запуск
    }
}